core/scheduler/service/job_run_service.go

```diff
@@ -230,6 +230,12 @@ func (s *JobRunService) registerNewJobRun(ctx context.Context, tenant tenant.Ten
 	if err != nil {
 		return err
 	}
+	telemetry.NewGauge("scheduler_events", map[string]string{
```

Comment:
the reason to use gauge earlier was because we wanted to restart the metric value as the generated metrics will be a lot and have it isolated. cmiiw @Mryashbhardwaj 

as part of this PR, we will no longer use `scheduler_events`, but instead segregate it to `job_run_events`, `sensor_run_events`, `task_run_events`, `hook_run_events`. each of the events will have either a job name label or an operator label (except sensor for now). we will use counter for this.